api/envoy/config/filter/network/redis_proxy/v2/redis_proxy.proto

```diff
@@ -50,4 +50,8 @@ message RedisProxy {
 
   // Network settings for the connection pool to the upstream cluster.
   ConnPoolSettings settings = 3 [(validate.rules).message.required = true];
+
+  // Indicates that latency stat should be computed in micro seconds. By default it is computed in
+  // milli seconds.
+  bool latency_in_micros = 4;
```

Comment:
Would it make sense to have this configuration more generic i.e. allow for specifying the time unit and produce stats accordingly?

Reply:
@venilnoronha  Since redis is a low latency application in almost all the cases either the stats will be in microseconds or milliseconds.  IMO making the configuration to allow time unit will make it bit cumbersome for end user to specify it as a config.